R/mod_review_forms.R

```diff
@@ -131,16 +172,14 @@ mod_review_forms_server <- function(
         # it will give a warning. This would be rare since it would mean a datapoint with the same edit date-time was reviewed but another one was not. 
         # probably better to use defensive coding here to ensure the app does not crash in that case. However we need to define which review status we need to select
         # in this case get the reviewed = "No"
-        review_status <- with(review_data_active(), reviewed[edit_date_time == max(as.POSIXct(edit_date_time))]) |> unique()
-        review_comment <- with(review_data_active(), comment[edit_date_time == max(as.POSIXct(edit_date_time))]) |> unique()
-        if(length(review_status) != 1) warning("multiple variables in review_status, namely: ", 
-                                               review_status, "Verify data.")
+        review_status <- unique(review_data_active()[["reviewed"]])
```

Comment:
I think `review_status` can now contain both `TRUE` and `FALSE`, based on differences in edit_date_time. I this intended? Or is this not true?

Reply:
It can contain both "Yes" and "No" and that is intentional. It is no longer attached to `edit_date_time`, but instead grabs the values directly from the `reviewed` field of the active review data. We no longer need `edit_date_time` because records are now unique in the database.